lib/i18n_yaml_editor/web.rb

```diff
@@ -41,7 +41,11 @@ def app
       on post, "update" do
         if translations = req["translations"]
           translations.each {|name, text|
-            app.store.translations[name].text = text
+            if text == '__delete__'
```

Comment:
And what we're doing is deleting a key AND it's translations, not just the translation.